<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>477</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>477</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>textInput</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: #f6e58d;</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>431</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
border-radius: 15px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;이순신 돋움체 M&quot;;
color: #3867d6;</string>
   </property>
   <property name="text">
    <string>텍스트를 입력하세요!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="textCheck">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
border-radius: 20px;
background-color: rgb(255, 255, 255);
color: #3867d6;</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
